File: Assets/MainCam.js
#pragma strict

var Scores : Transform;

function Start () {
/*
	var scorePosition : float = Scores.localPosition.x;
	Debug.Log(Scores.localPosition.y);
	for (var j = 0; j < 15; j++) {
		scorePosition += 3f;
		Instantiate (Scores, Vector3(scorePosition, 16f, 0f), Quaternion.identity);
	}
	*/
}

function Update () {
	rigidbody2D.velocity.x = xSpeed;
}

File: Assets/MainCam.cs
using UnityEngine;
using System.Collections;

public class MainCam : MonoBehaviour {

	// Use this for initialization
	void Start () {
	
	}
	
	// Update is called once per frame
	void Update () {
		rigidbody2D.velocity = new Vector3 (variable.xSpeed, 0, 0);
		// rigidbody2D.velocity.x = variable.xSpeed;
	}
}
